README.md

```diff
@@ -103,6 +103,14 @@ However, in many cases you may need to customize the stack to match your organiz
 best practices. See [the stack customization guide](stack-customization.md)
 for details on how to do this.
 
+### Does the MLOps stack cover data (ETL) pipelines?
+While some components of the MLOps stack can be re-used and customized for ETL pipelines,
+the focus of the Databricks MLops stack on productionizing ML workflows (training, model deployment, batch inference, etc.).
+You might find the following resources more useful for data processing: [dbx](https://docs.databricks.com/dev-tools/dbx.html) 
```

Comment:
Should we also mention Databricks feature store? Or do you wanna wait on that

Reply:
Done (we can update more as follow-ups).